src/keys/get-management-token.ts

```diff
@@ -67,25 +70,45 @@ const getTokenFromOneTimeToken = async (
     `Successfully retrieved CMA Token for app ${appInstallationId} in space ${spaceId} and environment ${environmentId}`
   )
 
-  return JSON.parse(response.body).token
+  return JSON.parse(response.body).token as Promise<string>
```

Comment:
`JSON.parse` does not return a `Promise` 🤔

Reply:
Accidentally applied to the return statement instead of the function type (it's async, so it returns a Promise, but obviously JSON.parse doesn't)